docs/ReleaseNotes.md

```diff
@@ -1,22 +1,75 @@
 # Acuminator Release Notes
-This document provides information about fixes, enhancements, and key features that are available in Acuminator 1.2.
+This document provides information about fixes, enhancements, and key features that are available in Acuminator 1.3.
+
+## Acuminator 1.3
+Acuminator 1.3 includes the diagnostics and code fixes, enhancements, and bug fixes described in this section, as well as the features that have been implemented in the previous versions.
+### New Diagnostics and Code Fixes
+In this version, diagnostics and code fixes for the following issues have been added.
+
+| Code   | Issue Description                                               | Type    | Diagnostics | Code Fix  | 
+| ------ | --------------------------------------------------------------- | ------- | ----------- | --------- | 
+| PX1012 | `PXAction` is declared on a non-primary view.                     | Warning | Available   | Available |
+| PX1015 | For a BQL statement that contains only `Required` parameters, the number of arguments of the `Select` method must be equal to the number of `Required` parameters. | Warning | Available | Unavailable |
```

Comment:
@k-s-popova, the PX1015 message is not exactly valid. It seems that it was taken from the JIRA item which has outdated message. The Acuminator's wiki page is more precise. Currently the BQL statement can contain all types of parameters - Current,  Required and Optional. The supported methods are not only "Select" but also other similar methods.

Reply:
@SENya1990 , the wiki page has nothing about Current and Optional. Please update.

Reply:
@k-s-popova, yes it was outdated too. Updated the wiki.